runtime/src/lib.rs

```diff
@@ -456,13 +456,8 @@ parameter_types! {
 impl pallet_treasury::Config for Runtime {
     type PalletId = TreasuryPalletId;
     type Currency = Balances;
-    type ApproveOrigin = EitherOfDiverse<EnsureRoot<AccountId>, Treasurer>;
```

Comment:
What would be the track in the referenda pallet ? "Treasurer" ?

Reply:
Maybe it could be worth adding a test showcasing the new flow ?

Reply:
Actually digging a bit more, I found [this](https://github.com/paritytech/polkadot-sdk/issues/138 ) as well

```The old treasury pallet's dispatchables to create/reject/approve a proposal now replaced by the spend dispatchable.

Deprecated dispatchables:

pallet_treasury.propose_spend
pallet_treasury.reject_proposal
pallet_treasury.approve_proposal
```

The comment on the PR refers to a point time even before removing the ApproveOrigin type 

Also `spend` in the new version is defined as  "Propose and approve a spend of treasury funds, allows spending any asset kind managed by the treasury"

bottom line, yes we could add a new test to call `spend` or `spend_local` and yes I think the track is Treasurer